luigi/__init__.py

```diff
@@ -48,3 +48,14 @@
 from luigi.event import Event
 
 from .tools import range  # just makes the tool classes available from command line
+
+
+__all__ = [
```

Comment:
Documentation will describe better then me.

https://docs.python.org/2/tutorial/modules.html#importing-from-a-package

One the other side my motivation was remove warnings like:

```
luigi/__init__.py:34:1: F401 'Parameter' imported but unused
```